store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9640,4 +9640,26 @@ TODO: delete them permanently from here
   <globalConfigValue>1518163473</globalConfigValue>
 </attr>
 
+<attr id="3024" name="zimbraFeatureResetPassword" type="boolean" cardinality="single" optionalIn="account,cos,domain" flags="accountCosDomainInherited" since="8.8.9">
```

Comment:
Please make it zimbraFeatureResetPasswordEnabled

Reply:
Done